index.js

```diff
@@ -9,10 +10,26 @@ function CoffeeScriptFilter (inputTree, options) {
   Filter.call(this, inputTree, options)
   options = options || {}
   this.bare = options.bare
+  this.options = options;
 }
 
 CoffeeScriptFilter.prototype.extensions = ['coffee', 'litcoffee', 'coffee.md']
 CoffeeScriptFilter.prototype.targetExtension = 'js'
+CoffeeScriptFilter.prototype.baseDir = function() {
+  return __dirname;
```

Comment:
When `options.persistent = true` is provided (it may want to be enabled by default) this code-path is taken.

persistent mode is opt-in, but the tests should stress it.